tests/fuzz/fuzz_client.c

```diff
@@ -0,0 +1,227 @@
+#include "mbedtls/ssl.h"
+#include "mbedtls/entropy.h"
+#include "mbedtls/ctr_drbg.h"
+#include "mbedtls/certs.h"
+#include <string.h>
+#include <stdlib.h>
```

Comment:
`random` and `srandom` require `_XOPEN_SOURCE >= 500` or `_XOPEN_SOURCE && _XOPEN_SOURCE_EXTENDED`. Please define these explicitly in the file so as not to depend too much on specific build infrastructure. Applies to all the files where `random` is used.

Alternatively, can you use `srand` and `rand`?

Reply:
ok I use `rand` and `stand`